readme.md

```diff
@@ -5,6 +5,37 @@
 Warp brings Solidity to StarkNet, making it possible to transpile Ethereum
 smart contracts to StarkNet Cairo Contracts.
 
+## Quickstart
+
+Docker compose provides a ready to use environment featuring warp and devnet.
+
+### Build and run containers
+
+```bash
+docker-compose up
+```
+
+### Transpile
+
+```bash
+docker-compose exec warp warp transpile example_contracts/ERC20.sol
+```
+
+Alternatively for warp developers:
+
+```bash
+docker-compose exec warp npx ts-node src transpile example_contracts/ERC20.sol
+```
+
+It's best to copy the contract/repo to the warp directory so it is available in container via volume. Use contract's paths relative to warp root. For example, assuming you've copied your project to `warp/projects/myproject` you can replace `example_contracts/ERC20.sol` with `projects/myproject/mycontract.sol` in the above commands.
+
+### Deploy to devnet
```

Comment:
We should move this to the dev section, there is a command for compiling with warp
`warp compile` we should use that.

In a follow up PR can you also make it possible to deploy to the devnet using warp deploy and then replace starknet deploy with warp deploy

Reply:
Not sure what you mean by dev section. If [that](https://github.com/NethermindEth/warp#warp-installation-method-2-from-sourcefor-devs) then the section is not describing docker-compose usage